students/21th/mingikim/js/main.js

```diff
@@ -0,0 +1,63 @@
+const commentInput = document.querySelector(".input-comment");
+const commentList = document.querySelector(".comment-list");
+const commentButton = document.querySelector(".article-input__button");
+
+function paintComment(text) {
+  const li = document.createElement("li");
+  const delBtn = document.createElement("button");
+  const likeBtn = document.createElement("button");
+  const commentBox = document.createElement("div");
+  const bottonBox = document.createElement("div");
+  delBtn.innerText = "✖";
+  likeBtn.innerText = "❤";
+  li.innerHTML = text;
+  delBtn.classList.add("delete-button");
+  delBtn.addEventListener("click", () => {
+    commentList.removeChild(commentBox);
+  });
+  likeBtn.classList.add("like-button");
+  likeBtn.addEventListener("click", () => {
+    likeBtn.classList.toggle("like");
+  });
+  commentBox.classList.add("comment-box");
+  bottonBox.classList.add("botton-box");
+  bottonBox.appendChild(delBtn);
+  bottonBox.appendChild(likeBtn);
+  commentBox.appendChild(li);
+  commentBox.appendChild(bottonBox);
+  commentList.appendChild(commentBox);
+}
+
+function handleClick() {
+  paintComment(commentInput.value);
+  commentInput.value = "";
+  commentButton.disabled = true;
+}
+
+function useButton(event) {
+  const inputedValue = event.target.value;
+  inputedValue
+    ? (commentButton.disabled = false)
+    : (commentButton.disabled = true);
```

Comment:
boolean의 특성을 이용해서
```suggestion
  commentButton.disabled = !inputValue
```
로 표현 가능해보입니다!

Reply:
수정 완료 했습니다.
참고가 많이 됬습니다!